resources/CONTAINER_VERSION

```diff
@@ -0,0 +1 @@
+LATEST=v13
```

Comment:
It would be easier to create a script that parses it if we just write `13` instead of `LATEST=13`.

Reply:
Updated. I added `LATEST` to make it clearer, but if it's not necessary, we can have only the number here.